Drive/rover_drive_system.hpp

```diff
@@ -127,17 +143,27 @@ class RoverDriveSystem
     sjsu::LogInfo("back wheel position: %f", back_wheel_.GetPosition());
   };
 
-  /// Parses incoming data from mission control to command rover
-  /// @param response incoming JSON / string data from mission control
-  void ParseMissionControlData(char * response)
+  /// Parses incoming JSON data from mission control to command rover
+  /// @return true if successfully parsed with correct number of cmds
+  bool ParseMissionControlResponse()
   {
-    // TODO: parse data using sscanf. Refer to GitHub Issue #152 for solution
-    sjsu::LogInfo("MISSION CONTROL RESPONSE:\n%s", response);
-    // hard coded for now
-    mission_control_data_.is_operational = true;
-    mission_control_data_.drive_mode     = 'S';
-    mission_control_data_.rotation_angle = 10.0f;
-    mission_control_data_.speed          = 30.0f;
+    const int expected_num_cmds = 4;
+    int parsed_num_cmds =
+        sscanf(mission_control_data_.response,
+               "{ \"is_operational\": %d, \"drive_mode\": "
+               "\"%c\", \"speed\": %f, \"angle\": %f}",
```

Comment:
Rather than using all of these escape characters you can use raw strings.

Reply:
I didn't know this was a thing lol. My initial concern is whether the format specifiers will still work? I'll try it regardless